src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -72,11 +72,14 @@ export class RelayFeeCalculator {
   }
 
   async gasFeePercent(amountToRelay: BigNumberish, tokenSymbol: string): Promise<BigNumber> {
-    const [gasCosts, tokenPrice, decimals] = await Promise.all([
-      this.queries.getGasCosts(tokenSymbol),
-      this.queries.getTokenPrice(tokenSymbol),
-      this.queries.getTokenDecimals(tokenSymbol),
-    ]);
+    const getGasCosts = this.queries.getGasCosts(tokenSymbol).catch((error) => {
```

Comment:
does this .catch() combination into Promise.all() work? never seen this before but I've never tried

Reply:
I think so from googling around :) catch() can't be added to Promise.all() as that would obscure which call the error comes from specifically